### ISSUE TITLE
Some weights seem off.

### ISSUE THREAD

**Issue by Nhatorama** [CleverRaven/Cataclysm-DDA#18926]
**survivor machete  0.94 lbs**
switchblade 1.02 lbs

**makeshift machete 1.19 (needs a blade to be crafted)**
blade 1.22 lbs

**Comment by Rivet-the-Zombie** [comment]
Unless something has changed, that's the weight of a stack of five throwing knives.

**Comment by Rivet-the-Zombie** [comment]
Same with the stack of throwing axes.

**Comment by Nhatorama** [comment]
I haven't used them yet. I assumed it was the weight for a single throwing knife/axe.
The throwing stick(s) is a stack of 4.

My fault. I guess I have a good reason to use them now :D

**Comment by illi-kun** [comment]
@Nhatorama can you update your initial comment to keep only actual info?

**Comment by Nhatorama** [comment]
It would be helpful if the description stated how many throwing knife/axe/stick are made.

**Comment by illi-kun** [comment]
For what?

**Comment by illi-kun** [comment]
Uh, misread your note. OK, it probably makes sense.

**Comment by illi-kun** [comment]
I've tried to use ```"copy-from": "blade"``` in ```makeshift machete``` definition to have better match of weight, and ```proportional weight: 1.04``` works great for this but it seems to be impossible to supersede quality of blade (```[ "BUTCHER", -38 ]```) by better quality of makeshift machete ```[ "BUTCHER", -27 ]```. Both ```extend``` and ```delete``` don't work properly in this case.

**Comment by illi-kun** [comment]
@mugling can you take a look on this? I guess ```"copy-from"``` should be expanded to allow to replace value of ```"BUTCHER"``` modifier.

**Comment by mugling** [comment]
@illi-kun yes

**Comment by mugling** [comment]
Could you not do direct replacement via re-specifying `qualities` on the child item?

**Comment by mugling** [comment]
```
  {
    "id": "makeshift_machete",
    "copy-from": "blade",
    "type": "GENERIC",
    "category": "weapons",
    "name": "makeshift machete",
    "description": "This is a large blade that has had a portion of the handle wrapped in duct tape, making it easier to wield as a rough machete.",
    "volume": 8,
    "to_hit": 1,
    "proportional": {
        "weight": 1.05,
        "cutting": 2.0
    }
    "qualities": [ [ "CUT", 1 ], [ "BUTCHER", -27 ] ],
    "flags": [ "SHEATH_SWORD" ]
  }
```

**Comment by illi-kun** [comment]
@mugling I tried that earlier and it doesn't work (```duplicated quality``` error thrown by ```Item_factory::set_qualities_from_json()```).

**Comment by mugling** [comment]
probably needs a call to `clear()` before trying to set the new qualities. I can fix that but it's a merge conflict with #19058. If you or anyone else has time to check that out it's a conceptually simple pure refactor. I'll PR a fix to this issue as soon as #19058 goes through.

**Comment by illi-kun** [comment]
Sorry, it's 4 a.m. in my timezone so I'm going to take a break. I'll take a look on it tomorrow if no one beats me on this.
